lib/rpc-client.js

```diff
@@ -36,7 +41,7 @@ export async function rpc(method, params, { rpcUrl, rpcAuth, fetch = globalThis.
     }),
   })
   const res = await fetch(req, {
-    signal: AbortSignal.timeout(60_000),
+    signal,
```

Comment:
Please add a TODO item to https://github.com/CheckerNetwork/roadmap/issues/250 to remind us to configure `signal: AbortSignal.timeout(60_000)` in the place that's looking up the miner's index-provider peer id. (I want to preserve the timeout behaviour.)